install.sh

```diff
@@ -576,6 +576,11 @@ if [[ "${#mkdirs[@]}" -gt 0 ]]; then
   execute_sudo "$CHGRP" "$GROUP" "${mkdirs[@]}"
 fi
 
+directories=(bin etc lib sbin)
+for dir in "${directories[@]}"; do
+  execute_sudo "/bin/chmod" "+t" "${HOMEBREW_PREFIX}/${dir}"
```

Comment:
Can you add a wee comment explaining what the sticky bit does and why we're setting it?